tests/docker/ducktape-deps/kgo-verifier

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
-git -C /opt clone https://github.com/redpanda-data/kgo-verifier.git
+git -C /opt clone https://github.com/NyaliaLui/kgo-verifier.git
```

Comment:
I'm using my fork to test the SHA. Once tests pass here, then we can merge the PR on kgo-verifier side at https://github.com/redpanda-data/kgo-verifier/pull/40